lib/web/plug.ex

```diff
@@ -0,0 +1,61 @@
+defmodule Xcribe.Web.Plug do
+  @moduledoc """
+  Server generated API documentation.
+
+  Add a doc scope to your router, and forward all requests to `Xcribe.Web.Plug`
+
+  ```
+        scope "doc/swagger" do
+          forward "/", Xcribe.Web.Plug
+        end
+
+  ```
+  """
+
+  use Plug.Router
+
+  alias Plug.Conn
+  alias Xcribe.Config
+
+  plug(Plug.Static, at: "/", from: :xcribe)
+  plug(:match)
+  plug(:dispatch)
+
+  @template [Path.dirname(__ENV__.file), "template.eex"] |> Path.join() |> File.read!()
```

Comment:
```
EEx.function_from_file(:defp, :template, Application.app_dir(:xcribe, "template.eex"), [:file, :uri])

# usage
template(conn.assigns.file, uri)
```

WDYT of evaluating the eex at compile with [`function_from_file`](https://hexdocs.pm/eex/EEx.html#function_from_file/5)? 

Also, WDYT of storing the template at the `priv`. Feels strange to me store it on the lib folder.

Reply:
Done :+1: